src/vm/zapsig.cpp

```diff
@@ -1186,7 +1190,9 @@ BOOL ZapSig::EncodeMethod(
     }
 
 #ifdef FEATURE_READYTORUN_COMPILER
-    if (IsReadyToRunCompilation())
+
+    // FUTURE: This condition should likely be changed or reevaluated once support for smaller version bubbles is implemented.
+    if (IsReadyToRunCompilation() && !IsLargeVersionBubbleEnabled())
```

Comment:
>IsLargeVersionBubbleEnabled [](start = 38, length = 27)

I think this still isn't right. It should be something like IsReadyToRunCompilation() && (!IsLargeVersionBubbleEnabled() ||| !pMethod-.GetModule()-.IsInCurrentVersionBubble())

Reply:
Changed.